src-cr/Components/Map.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import GoogleMapReact from 'google-map-react';
+
+
+const Marker = ({ text }) =>
+<div>
```

Comment:
This indentation is wrong, you can use a linter like [Eslint](https://eslint.org/) to write more consistent code. I usually recommend to use it with the Airbnb rules or [create-react-app](https://www.npmjs.com/package/eslint-config-react-app) rules. 

Also checkout https://github.com/prettier/prettier I use it as an Atom plugin.